app/models/package_manager/packagist.rb

```diff
@@ -105,7 +105,7 @@ def self.acceptable_versions(versions)
     end
 
     def self.dependencies(_name, version, mapped_project)
-      vers = mapped_project[:versions][version]
+      vers = mapped_project[:versions].find { |v| v["version"] == version }
```

Comment:
Is this packagist specific?

Reply:
yeah, this is specifically for the Packagist project response